front/package.json

```diff
@@ -13,4 +13,4 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
   }
-}
\ No newline at end of file
+}
```

Comment:
Where are redux and redux-saga dependencies? Does not compile after git pull.

Reply:
Yes, you're right, i improve that